scripts/polynomial_perf_regression.rb

```
@@ -0,0 +1,149 @@
+#!/usr/bin/env ruby
```

RuboCop (1.85.1)

[warning] Script file polynomial_perf_regression.rb doesn't have execute permission.

(Lint/ScriptPermission)